python/tvm/relay/frontend/tensorflow_parser.py

```diff
@@ -0,0 +1,153 @@
+"""TF: Tensorflow parser"""
+from __future__ import absolute_import as _abs
+from __future__ import print_function
+import os
+from tensorflow.core.framework import graph_pb2
+from tvm.contrib import util
+
+
+class TFParser(object):
+    """A Wrapper to handle tensorflow models parsing
+       TensorFlow is needed
+    ```
+    parser = TfParser(model_dir)
```

Comment:
Nit: move this code to an example section?

Examples
---------
```
.. code-block:: python
    parser = TfParser(model_dir)
    graph = parser.parse()
```

Reply:
added, thx.